src/components/colors/Colors.stories.tsx

```diff
@@ -31,12 +31,13 @@ const renderColorPalette = (colorEntry: [string, ColorEntry], index: number) =>
   return (
     <div key={index}>
       <h2 style={{ marginTop: '0' }}>{colorName}</h2>
-      {Object.entries(colorVariations).map((colorVariationEntry, index) => {
+      {Object.entries(colorVariations).map((colorVariationEntry, i) => {
```

Comment:
Not a blocker, but I'm definitely a fan of being more verbose and just writing out "index".

Reply:
In this case there is already an `index` defined on line 32, so lint was complaining.